drivers/scheduler/rke/rke.go

```diff
@@ -44,20 +45,26 @@ func (r *Rancher) String() string {
 func (r *Rancher) Init(scheduleOpts scheduler.InitOptions) error {
 	var err error
 	err = r.K8s.Init(scheduleOpts)
-	rkeParametersValue, err := r.GetRancherClusterParametersValue()
 	if err != nil {
 		return err
 	}
-	rancherClientOpts := rancherClientBase.ClientOpts{
-		URL:       rkeParametersValue.Endpoint,
-		TokenKey:  rkeParametersValue.Token,
-		AccessKey: rkeParametersValue.AccessKey,
-		SecretKey: rkeParametersValue.SecretKey,
-		Insecure:  true,
-	}
-	r.client, err = rancherClient.NewClient(&rancherClientOpts)
-	if err != nil {
-		return err
+	// This is needed only in case of px-backup as other platforms do not create new RKE client
+	if flag.Lookup("backup-driver").Value.(flag.Getter).Get().(string) == "pxb" {
```

Comment:
Minor: Use the const for `pxb` instead of string

Reply:
done